pkgs/misc/screensavers/electricsheep/default.nix

```diff
@@ -1,22 +1,51 @@
-{stdenv, fetchurl, pkgconfig, expat, zlib, libpng, libjpeg, xorg}:
+{ stdenv,
+  fetchgit,
+  pkgs,
+  #autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
-  name = "electricsheep-2.6.8";
+  name = "${pname}-${version}";
+  pname = "electricsheep";
+  version = "2.7b33-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "598d93d90573b69559463567540aac8bc8a5b3f3";
   
-  src = fetchurl {
-    url = "http://electricsheep.org/${name}.tar.gz";
-    sha256 = "1flqcqfs75wg74hr5w85n6w8b26l4qrpwzi7fzylnry67yzf94y5";
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/scottdraves/${pname}.git";
+    sha256 = "1zcn6q0dl0ip85b8b4kisc5lqjb1cs0hpzlx4l5995l6mhq9kxis";
   };
 
-  buildInputs = [pkgconfig expat zlib libpng libjpeg xorg.xlibsWrapper xorg.libXv];
+  #nativeBuildInputs = [ autoreconfHook ];
```

Comment:
currently, i am running `./autogen.sh` in `configurePhase` manually, as `autoreconfHook` is giving me the following error:

`./configure: line 18537: syntax error near unexpected token PTHREAD_OK=true,PTHREAD_OK=false'`
`./configure: line 18537:ACX_PTHREAD(PTHREAD_OK=true,PTHREAD_OK=false)'`

im not sure about this error, or if im missing something when invoking `autoreconfHook` ?

It works correctly as is manually, without invoking `autoreconfHook`.  i left it commented to show this, as hopefully there is an obvious fix?